_benchmark/user-guide/optimizing-benchmarks/expand-data-corpus.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Expand data corpus
+nav_order: 20
+parent: Optimizing benchmarks
+grand_parent: User guide
+---
+
+# Expanding the data corpus of a workload
+
+This tutorial shows you how to use the [`expand-data-corpus.py`](https://github.com/opensearch-project/opensearch-benchmark/blob/main/scripts/expand-data-corpus.py) script to increase the size of the data corpus for a OpenSearch Becnhmark workload. This can help assist in running the `https_logs` Benchmark with a larger scale, for instance, with clusters containing multiple data nodes.
+
+Only the `http_logs` workload is currently supported.
+{: .warning}
+
+## Prerequisites
+
+To use this tutorial, make sure you fulfill the following prerequsities:
+
+1. Python 3.x or greater installed.
+2. The `http_logs` workload data corpus already downloaded in your cluster.
```

Comment:
I think this needs to be data corpus already downloaded in your load generation host (where OSB is running).